### PR TITLE
Add package name and version to User-Agent

### PR DESCRIPTION
**Pull request by bhelx**
This follows the Recurly convention and will help us with analytics
on the backend. Having this insight is necessary to moving this library
forward with confidence. I tested this by making a request and checking our logs.

\cc @mrfelton

### CODE REVIEW

File: lib/recurly-data.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const _ = require('lodash')
 const rparser = require('./parser.js')
 const debugR = require('debug')('recurring:request')
 const debug = require('debug')('recurring')
+const pkg = require('../package.json')
```
Comment (bhelx) [inline lib/recurly-data.js]: @mrfelton perhaps you know of a better way to do this?

**Comment by bhelx** [comment]
It appears that the integration tests are failing because this is an empty array:

https://github.com/ceejbot/recurring/blob/master/test/test-02-api.js#L694

It's failing on master as well

File: lib/recurly-data.js (hunk 1 above)
Comment (mrfelton) [inline lib/recurly-data.js]: @bhelx I know there is https://github.com/indexzero/node-pkginfo which is designed specifically for this sort of thing. I question wether the overhead of an extra dependency is worth it though. The way you have done it is pretty standard.

**Comment by mrfelton** [comment]
@bhelx I noticed this test failing a while back but haven't had the time to look into it. Would love a PR to get that fixed and get the build green again :)

It's unrelated to this PR though, so I went ahead and merged through. Thanks.

**Comment by bhelx** [comment]
@mrfelton sure I can take a look. It seems like that test is expecting there to be at least one `open` invoice when it's run and there aren't any. I can probably just create a manual invoice and leave it open in the setup block.

**Pull request merged**